client/store/user.js

```diff
@@ -6,6 +6,7 @@ import history from '../history'
  */
 const GET_USER = 'GET_USER'
 const REMOVE_USER = 'REMOVE_USER'
+const EDIT_USER = 'EDIT_USER'
```

Comment:
I don't think we need an EDIT_USER action type, it does the same thing has GET_USER. we should be able to use getUser instead